pom.xml

```diff
@@ -1170,5 +1193,19 @@ under the License.</licenseText>
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>pmd</id>
```

Comment:
Spot is configured in parent so how to do here? Profile is not working by default.